app/views/pages/_sidebar.html.erb

```diff
@@ -1,13 +1,23 @@
 <div class="container">
   <h2 style="text-align: center">All Pages</h2>
   <ul class="nav nav-pills nav-fill flex-column">
-    <%- pages.each do |page| %>
-        <%- is_current = page == current_page %>
+    <%- categories.each do |key, value| %>
       <li class="nav-item">
-        <a class="nav-link <%= "active" if is_current %>"
-          <%= "aria-current=\"page\"" if is_current %>
-          href="<%= page_path(page) %>"><%= page.title %>
-        </a>
+        <a class="nav-link font-weight-bold" data-toggle="collapse" 
+          href="#category-<%= key %>" role="button" aria-expanded="false" aria-controls="category-<%= key %>"><%= key %> </a>
```

Comment:
`key` needs some method to make it html safe. `key.parameterize` which makes it suitable for using in IDs and class names.